packages/@vue/cli/lib/ui.js

```diff
@@ -36,7 +36,7 @@ async function ui (options = {}, context = process.cwd()) {
     subscriptionsPath: '/graphql',
     enableMocks: false,
     enableEngine: false,
-    cors: '*',
+    cors: host,
```

Comment:
Shouldn't this be a `corsOptions` object?
https://github.com/apollographql/apollo-server/blob/e7949d62c0f2a3f096573f1ee07f146d7aabd8af/packages/apollo-server-express/src/ApolloServer.ts#L27